apps/condo/domains/notification/adapters/hcmAdapter.js

```diff
@@ -70,7 +74,7 @@ class HCMAdapter {
      * @returns {*}
      */
     static getAppType (appIds, token) {
-        return HUAWEI_APP_TYPE_BY_APP_ID[appIds[token]]
+        return HUAWEI_APP_TYPE_BY_APP_ID[get(appIds, token)]
```

Comment:
Actually appIds is generated and it should contain token if it was not modified somehow somewhere in between. But this way it will be a bit more stable.